### PR TITLE
Moving system info to rancher repo

### PR DESCRIPTION
**Pull request by mattmattox**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -66,3 +66,23 @@ steps:
       event:
         include:
         - push
+
+  - name: docker-build--system-info
+    image: plugins/docker
+    settings:
+      repo: rancher/system-info
```
Comment (dnoland1) [inline .drone.yml:73]: Should this be systems-info, not system-info? The references in the chart use systems-info.

Comment (mattmattox) [reply .drone.yml]: Fixed

**Review by dnoland1 — APPROVED** [review]
LGTM, merging so we can test drone

**Comment by dnoland1** [comment]
Looks like drone failed after merging - https://drone-pr.rancher.io/rancherlabs/support-tools/21 @mattmattox can you have a look?

**Pull request merged**